ansible/roles/test/templates/exabgp/start.j2

```diff
@@ -3,12 +3,12 @@ ifconfig eth{{ '%d' % (minigraph_port_indices[minigraph_vlans[minigraph_vlan_int
 ifconfig eth{{ '%d' % (minigraph_port_indices[minigraph_vlans[minigraph_vlan_interfaces[0]['attachto']]['members'][0]])}}:1 {{item.logical_ip_2}}
 
 {% set intf = 'eth%d' % (minigraph_port_indices[minigraph_vlans[minigraph_vlan_interfaces[0]['attachto']]['members'][1]]) %}
-ifconfig {{intf}} {{item.mux_ip_1}}
-i=0; until [$i -eq 10] || ping {{minigraph_vlan_interfaces[0]['addr']}} -I {{intf}} -c 1 >/dev/null 2>&1; do i=`expr $i + 1`; done &
+ifconfig {{intf}} add {{item.mux_ip_1}}
```

Comment:
Can you elaborate more why this change is needed?

My understanding is that without 'add', the interface IP will be replaced with the new one. With 'add', the new IP will be added on as secondary IP(s) on the interface.

What is the reason that we need to keep multiple IPs on the interface?

Reply:
in case runing in IPv6 mode 'ifconfig' doesn't expect to get arguments like
```
ifconfig eth2 fc00::3
```
it should be 
```
ifconfig eth2 inet6 add <addr>
or just
ifconfig eth2 add <addr>
```
i tried both and both are working. the only notes for the first one - it is required to pass `addr_family` from caller. e.g.
```
bgp_speaker.yml

    - name: Generate start file for exabgp instances
      template: src=roles/test/templates/exabgp/start.j2 dest={{exabgp_dir}}/{{item.file_name}} mode=u+rwx
      with_items:
        - {file_name: 'start.sh',
           addr_family: '{{addr_family}}',
           config_file_1: 'config_1.ini',
           config_file_2: 'config_2.ini',
           config_file_3: 'config_3.ini',

start.j2

ifconfig {{intf}} {% if item.addr_family=='ipv6' %} inet6 add{% endif %} {{item.mux_ip_1}}
```

Reply:
@pyadvichuk Thanks for the quick response.

I suspected so.

Does this change break IPv4 BGP speaker test? Does it put the system in a state that next test would fail?

Reply:
let me check once again. I've started both v4 and v6 many times in random sequence during investigation. not sure if I really tested v4 after the final fix.

Reply:
thanks! You might want to also run another test like fdb or fib to make sure other tests would pass afterwards too.

Reply:
in a sugested form  `ifconfig eth2 add <addr>` - `ifconfig` will not add secondary address but, simple set new one. so, for IPv4
```
ifconfig eth2 add <addr>
and 
ifconfig eth2 add <addr>
```
are equivalent

any way, will check tomorrow once again

Reply:
Thanks for validating!